lib/uo_cli.rb

```diff
@@ -486,7 +502,7 @@ def self.create_reopt_scenario_file(existing_scenario_file)
         row['REopt Assumptions'] = 'multiPV_assumptions.json'
       end
       # write new file
-      CSV.open(File.join(existing_path, 'REopt_scenario.csv'), 'w') do |f|
+      CSV.open(existing_scenario_file, 'w') do |f|
```

Comment:
should you check that the REopt assumptions row is already present in the file here? If it is, don't overwrite it, right?  I think you are calling this create_reopt_scenario_file function in 2 different places, so there is the possibility of overwriting user configuration?

Reply:
Yes, yes I should check if the column already exists. 😳